config.yaml

```diff
@@ -40,6 +40,18 @@ options:
     description: >
       Amount of disk space to allocate to root disk for virtual machine runner. Positive integers
       with GiB suffix.
+  vm-disk-read:
+    type: string
+    default: 100MiB
+    description: >
+      Limit of disk read IO per second of the root disk for virtual machine runner. Positive integers
```

Comment:
Why MiB specifically? I don't see that limitation in the code. Also, the name of the config doesn't seem too obvious to me, I'd rename it to `vm-disk-read-limit` or something similar (same for the other config).

Reply:
I have updated the descriptions.